common/views/components/CardGrid/CardGrid.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+import {classNames, cssGrid} from '../../../utils/classnames';
+import ExhibitionPromo from '../ExhibitionPromo/ExhibitionPromo';
+import type {MultiContent} from '../../../model/multi-content';
+
+type Props = {|
+  items: MultiContent[]
+|}
+
+const CardGrid = ({
+  items
+}: Props) => (
+  <div className='css-grid__container'>
+    <div className='css-grid'>
+      {items.map(item => (
+        <div key={item.id} className={classNames({
+          [cssGrid({s: 12, m: 6, l: 4, xl: 4})]: true
```

Comment:
I don't know that we can assume cards will always appear in a 3-up configuration – feels like this is what's being suggested here?

Also, is this specific to ExhibitionPromos only? Seems kinda niche.

Reply:
This is t give us something to start with - and evolve.
It should be able to take all types of card, and currently on the homepage etc, we only use 3-up.

Trying to work on the process of making what we need, naming it, and seeing if we can evolve it or need something new.